dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -322,12 +322,10 @@ protected Map<String, Object> createDagDefine(User loginUser,
             logger.info("The task has not changed, so skip");
         }
         if (saveTaskResult == Constants.DEFINITION_FAILURE) {
-            putMsg(result, Status.CREATE_TASK_DEFINITION_ERROR);
```

Comment:
This needs to be returned to the front end, and can't be returned after deleting it?